_pages/dev/ssr/server-side-rendering-contextual-logging.md

```diff
@@ -6,32 +6,20 @@ feature:
   cx_version: n/a
 ---
 
-TL;DR How to fully enable the feature:
```

Comment:
`query:` Is the 'TL;DR' section forbidden? As we discussed with @Platonn, the document is quite long and from a customer perspective, would be great to have such (or a similar) section at the beginning of this doc that describes all steps that are necessary to fully enable contextual logging.

Reply:
IMHO it's a feature that every customer would like to start using from day 1.
But due to backward compatibility, the process of enabling this feature requires some manual steps.
So I think we should place all those steps somewhere at the very beginning of this document, so customers can start using it immediately.

In a future major release, some of those steps will implicit defaults, then this section will be much smaller. But then I think we will still keep a disclaimer with the meaning: "how not to break this feature: use LoggerService yourself, and make your 3rd party dependencies also use LoggerService - instead of native console"